teachopencadd/talktorials/T013_query_pubchem/talktorial.ipynb

```diff
@@ -6,10 +6,13 @@
    "source": [
```

Comment:
<p>Use f-strings?</p><p><code>print(f"The structure of Gefitinib:\n{Chem.MolFromSmiles(query)}")</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>It didn't work, only print a string.</p>

Reply:
<p>Oh right, that is my bad - forget about my suggestions :)</p>